docs/blog/2019-05-25-sentry-integration-with-gatsby/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Integrating Sentry Into Your Gatsby Project"
+date: "2019-05-25"
+author: "Jonathan Reeves"
+tags:
+  - error monitoring
+  - post
+  - sentry
+---
+
+## Assuming knowledge on my part
+
+I'm going to do something that you are told not to ever do and assume that you already have a project created and are just looking to install an error tracking software to help you fix and resolve errors before any users find it or at least before majority of your users find it and start complaining.
```

Comment:
this sentense `I'm going to do something that you are told not to ever do`:
- it is hard to understand what you mean with this as not native
- why it is a `bad thing` to do it

Reply:
I went ahead and removed that sentence and replaced it with a new one that make more sense.